docs/tutorials/setting-up-ethernet-over-usb-on-linux-command-line.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Setting Up Ethernet Over USB on Linux with Command Line
```

Comment:
Okay, The RaspberryPi doesn't have network-manager installed by default. 
When I find a persistent method then I will update the title accordingly.